cogs/tld.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+import voxelbotutils as utils
+
+
+class TldCommands(utils.Cog):
+
+    TLD_URL = "https://data.iana.org/TLD/tlds-alpha-by-domain.txt"
+
+    def __init__(self, bot:utils.Bot):
+        super().__init__(bot)
+        self.tld_list = None
+
+    async def load_tlds(self):
+        """Loads tlds into the cache"""
```

Comment:
Docstring wants to follow the same style as the rest of the project, ie the string goes on a line by itself ending with a period - ```py
"""
Loads TLDs into the cache.
"""```